src/act/node/param.rs

```diff
@@ -0,0 +1,65 @@
+use proc_macro2::TokenStream;
+
+use crate::{
+    act::node::{candid::type_annotation::ToTypeAnnotation, CandidType},
+    traits::ToIdent,
+};
+
+use super::{candid::type_annotation::TypeAnnotation, declaration::Declare, Declaration};
+
+#[derive(Debug, Clone)]
+pub struct Param {
+    pub name: String,
+    pub candid_type: CandidType,
+}
+
+impl Param {
+    pub fn get_prefixed_name(&self) -> String {
```

Comment:
Make this a trait with a default implmentat that evertything tha thas a prefixed name, ie, this and inint and heatbeat etc, can use to have unified prefixes

Reply:
we decided not to do this at this point, because we didn't want to take the time to think through the implications. We might come back to this later.